package.json

```
@@ -20,7 +20,7 @@
         ]
     },
     "scripts": {
-        "acceptancetest": "ts-mocha ./packages/server/tests/acceptance.spec.ts --exit",
+        "acceptancetest": "ts-mocha packages/server/tests/acceptance/rpc.spec.ts --exit",
```

Can make this pick up all the test files in this directory
```suggestion
        "acceptancetest": "ts-mocha packages/server/tests/acceptance/*.spec.ts --exit",
```